bin/Translation/translation_dumper.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Extract translation information from a JS file.
+ *
+ * Usage: pass the path of a JS file as a parameter.
+ *
+ * The script will output JSON containing the translation information on the standard output.
+ * Example of output:
+ * `{"translationsFound":[{"key":"mykey","domain":"mydomain"}]}`
+ *
+ */
+
+var walk = require( 'estree-walker' ).walk,
+    acorn = require( 'acorn' ),
+    fs = require('fs'),
+    sourceCode = fs.readFileSync(process.argv[2]),
+    ast = acorn.parse( sourceCode, {} ),
+    result = { translationsFound: [] };
+
+walk( ast, {
+  enter: function ( node, parent ) {
+
+        if( node.type && node.type == 'CallExpression' && node.callee.property.name == 'trans' ) {
+            result.translationsFound.push({
+                key: node.arguments[0].value,
```

Comment:
what happens if the key and/or domain parameters are variables or any other expression ?